source/MaterialXView/Viewer.cpp

```diff
@@ -200,6 +201,42 @@ Viewer::Viewer(const mx::StringVec& libraryFolders,
     _geometryHandler->loadGeometry(_searchPath.find(meshFilename));
     updateGeometrySelections();
 
+    _envGeometryHandler = mx::GeometryHandler::create();
+    _envGeometryHandler->addLoader(loader);
+    mx::FilePath envSphere("resources/Geometry/sphere.obj");
+    _envGeometryHandler->loadGeometry(_searchPath.find(envSphere));
+    const mx::MeshList& meshes = _envGeometryHandler->getMeshes();
+    if (!meshes.empty())
+    {
+        // Invert u and rotate 90 degrees.
+        mx::MeshPtr mesh = meshes[0];
+        mx::MeshStreamPtr stream = mesh->getStream(mx::MeshStream::TEXCOORD_ATTRIBUTE, 0);
+        mx::MeshFloatBuffer &buffer = stream->getData();
+        size_t stride = stream->getStride();
+        for (size_t i = 0; i < buffer.size() / stride; i++)
```

Comment:
We can't do a wrap via a transform easily so just do the math here to put the uvs in the correct place. The .25 is to handle the envmap lookup transform. The 1-u is done to reverse the uvs since we want to see the mirror of what's reflected.